packages/models/test/fixtures/report.mock.ts

```diff
@@ -6,7 +6,7 @@ import { lighthousePluginReport } from './lighthouse-plugin.mock';
 export function report(): Report {
   return {
     packageName: '@code-pushup/core',
-    version: '0.1.0',
+    version: '0.0.1',
```

Comment:
Why change it? :raised_eyebrow:

Reply:
It happened to be different… flakyness?

Reply:
This definitely shouldn't be flaky :sweat_smile: You changed the version in `packages/utils/src/lib/__snapshots__/report-to-md.spec.ts.snap` too. If you change them both back, I don't see why there should be an issue. If there is, it's probably something we need to know about.

Reply:
will be resolved with #174 followup and #180